web/docs/auth/auth-hooks.md

```diff
@@ -828,9 +888,31 @@ Wasp passes the `oauth` object to the `onAfterSignup` and `onAfterLogin` hooks o
 
 It has the following fields:
 
-- `accessToken: string`
+- `providerName: string`
+
+  The OAuth provider name (e.g. `'google'`, `'github'`) that the user authenticated with.
+
+- `tokens: Tokens`
+
+  You can use the OAuth tokens to make requests to the provider's API on the user's behalf.
+
+  Depending on the OAuth provider, the `tokens` object might have different fields. For example, for Google, it has the `accessToken`, `refreshToken`, `idToken`, and `accessTokenExpiresAt` fields.
+
+  <ShowForTs>
+
+  You can narrow down the `oauth.tokens` object type to the specific OAuth provider type to access the provider-specific fields:
```

Comment:
```suggestion
To access the provider-specific fields, you must first narrow down the `oauth.tokens` object type to he specific OAuth provider type.
```
Suggestions based on:
- Start with a problem, follow with a solution.
- can -> must (because they must :))